src/main/java/blusunrize/immersiveengineering/common/blocks/metal/TileEntityBelljar.java

```diff
@@ -255,6 +255,14 @@ public IPlantHandler getCurrentPlantHandler()
 		return curPlantHandler;
 	}
 
+	public boolean shouldGrow()
+	{
+		ItemStack soil = inventory.get(SLOT_SOIL);
+		ItemStack seed = inventory.get(SLOT_SEED);
+		IPlantHandler handler = getCurrentPlantHandler();
+		return (energyStorage.getEnergyStored() > IEConfig.Machines.belljar_consumption&&fertilizerAmount > 0&&handler!=null&&handler.isCorrectSoil(seed, soil)&&fertilizerAmount > 0);
```

Comment:
Shouldn't the energy check be for greater than _or equal to_ the minimum power needed for a run cycle? Doesn't really matter if the power stored drops down to zero, right? Also, why did you doubl the fertiliser check? It's the second and fifth ones.
```suggestion
		return (energyStorage.getEnergyStored() >= IEConfig.Machines.belljar_consumption&&fertilizerAmount > 0&&handler!=null&&handler.isCorrectSoil(seed, soil));
```